diff --git a/sqlparse/lexer.py b/sqlparse/lexer.py
index 8f88d17..92f04b5 100644
--- a/sqlparse/lexer.py
+++ b/sqlparse/lexer.py
@@ -134,6 +134,9 @@ class Lexer:
             raise TypeError("Expected text or file-like object, got {!r}".
                             format(type(text)))

+        with open("/mount/src/sqlparse/extract_testcases/extracted-sqlparse-testcases.txt", "a") as file:
+            file.write(text + "\n<end_of_query/>\n")
+
         iterable = enumerate(text)
         for pos, char in iterable:
             for rexmatch, action in self._SQL_REGEX:
